__index__.md

```diff
@@ -1,8 +1,10 @@
 
 - **[cache/v0.1](/cache/v0.1)** ([📄 graphql](cache/v0.1/cache-v0.1.graphql))
+- **[cache/v0.2](/cache/v0.2)** ([📄 graphql](cache/v0.2/cache-v0.2.graphql))
 - **[core/v0.1](/core/v0.1)** ([📄 graphql](core/v0.1/core-v0.1.graphql))
 - **[core/v0.2](/core/v0.2)** ([📄 graphql](core/v0.2/core-v0.2.graphql))
 - **[cost/v0.1](/cost/v0.1)** ([📄 graphql](cost/v0.1/cost-v0.1.graphql))
+- **[fakes/v0.0](/fakes/v0.0)** ([📄 graphql](fakes/v0.0/fakes-v0.0.graphql))
```

Comment:
I don't know the version schema but should we really be starting with v0.0? Shouldn't it at least be v0.1?

Reply:
Excellent question! I think we all have cognitive dissonance starting our versions at `1` and our arrays at `0` 😄 

It's even weirder because almost everyone makes version `1` start at `1.0` but not version `0` 🤷 . This is all wildly inconsistent IMO but I won't die on this hill. I'm down to keep the `0.1` tradition if you feel it's better.

Reply:
Just out of curiosity, what's the concern? Is it possible incomprehension from the users? Something else?

Reply:
While `v0.0` is technically valid it just feels off as a version number, right? Would you consume a package versioned as 0.0?

Reply:
I would not only consume it but also love that the maintainers use a consistent versioning scheme!

Agree I'm the minority though (for now!).

Reply:
To elaborate a bit more, I would consume it as a pre-release version, meaning basically draft status, API can change, etc... Just like any other `0.x` version. 

TBH I expect this version to change relatively soonish so even if there were additional warnings attached to `0.0`, I'd say it's probably desired here. After all, it's the very first release of something that has had very little usage in the field so it might as well come with extra warning signs.